.github/workflows/test.yml

```diff
@@ -52,6 +52,10 @@ jobs:
         elif [[ "${{matrix.BACKEND}}" == "jax" ]]; then
           echo "Installing JAX";
           pip install jax jaxlib;
+        elif [[ "${{matrix.BACKEND}}" == "paddle" ]]; then
+          echo "Installing PaddlePaddle";
+          pip install --pre paddlepaddle -i https://www.paddlepaddle.org.cn/packages/nightly/cpu/;
```

Comment:
Would it be possible to test against a released version? This would help to ensure TensorLy works for users.

Reply:
Updated to `3.0.0b1`